src/components/NewsletterForm/NewsletterFormEmail.tsx

```diff
@@ -32,6 +32,7 @@ export const NewsletterFormEmail = ({
         Your email address
       </label>
       <input
+        aria-describedby={hasError && `${id}-error`}
```

Comment:
This attribute expects an id or a space separated list of ids. If hasError is false then `false` would be output as the attribute value so I imagine the following would be better

```
aria-describedby={hasError ? /`${id}-error/` : null}
```